storey/flow.py

```diff
@@ -370,6 +382,51 @@ def join_from_response(element, response):
         JoinWithHttp.__init__(self, request_builder, join_from_response, **kwargs)
 
 
+def _build_request_put_records(partition_func, events):
+    record_list_for_json = []
+    for event in events:
+        record = event.element
+        record_as_bytes = None
+        if isinstance(record, bytes):
```

Comment:
what if the event is a dict?

Reply:
You need to serialize it first? :)
I guess I could add json.dumps in that case, to be nice.